src/App.jsx

```diff
@@ -1,13 +1,41 @@
 import * as React from 'react'
-import Button from './Button'
-import Pagination from './Pagination'
+import Card from './Card'
+import './App.css'
+
+const quizData = [
+  {
+    title: 'One big fat title!',
+    question: 'Was ist grün, frisch und klopft an deiner Tür?',
+    answer: 'Ein Klopfsalat',
+    isAnswerVisible: false,
+    isSelected: false,
+    id: '1',
+  },
+  {
+    title: 'Wow, another big fat title!',
+    question: 'Was ist gelb, laut und lebensgefährlich?',
+    answer: 'Eine Banone',
+    isAnswerVisible: true,
+    isSelected: false,
+    id: '2',
+  },
+]
 
 export default () => {
   return (
-    <div>
-      <Pagination isLeftDisabled text="1/34" />
-      <Pagination text="2/34" />
-      <Pagination isRightDisabled text="34/34" />
+    <div className="App">
+      {quizData.map(
+        ({ title, question, answer, isAnswerVisible, isSelected, id }) => (
+          <Card
+            title={title}
+            question={question}
+            answer={answer}
+            isAnswerVisible={isAnswerVisible}
+            isSelected={isSelected}
+            id={id}
```

Comment:
You need a `key` here otherwise React won't know whether the array changes or not. 
```suggestion
            id={id}
            key={id}
```